docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -148,6 +148,29 @@ oc get secret nova-migration-ssh-key || oc create secret generic nova-migration-
 rm -f id*
 cd -
 ----
++
+
+. If TLS Everywhere is enabled, set LIBVIRT_PASSWORD to match the existing {OpenStackShort} deployment password:
++
+----
+declare -A TRIPLEO_PASSWORDS
+TRIPLEO_PASSWORDS[default]="$HOME/overcloud-passwords.yaml"
+LIBVIRT_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' LibvirtTLSPassword:' | awk -F ': ' '{ print $2; }')
```

Comment:
The tests look fine to me but we'll still need to fix the docs. Rather than trying to do the encoding inside `oc apply` and deal with quote escaping, perhaps we could define another variable, something like:

```
LIBVIRT_PASSWORD_BASE64=$(echo -n "$LIBVIRT_PASSWORD" | base64)
```

And then use that variable in the `oc apply`? Just a suggestion.

Reply:
Fixed latest commit